sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+    * Understand and document a high level roadmap of projects within this space, including CNCF and non-CNCF projects. Identify gaps in project landscape.
+
+    * For projects that fall within the CNCF, perform health checks.
+
+    * Perform discovery of and outreach to candidate projects
+
+    * Help candidate projects prepare for presentation to the TOC
+
+    * Every CNCF project will be assigned to one suitable SIG by the TOC.
+
+#### End User Education (Outbound Communication)
+
+    * Provide up-to-date, high quality, unbiased and easy-to-consume material to help end users to understand and effectively adopt cloud-native technologies and practises within the SIG’s area, for example:
+
+        * White papers, presentations, videos, or other forms of training clarifying terminology, comparisons of different approaches, available projects or products, common or recommended practises, trends, illustrative successes and failures, etc.
+
+        * As far as possible, information should be based on research and fact gathering, rather than pure marketing or speculation.  
+
+#### End User Input Gathering (Inbound Communication)
+
+    * Gather useful end user input and feedback regarding expectations, pain points, primary use cases etc.
+
+    * Compile this into easily consumable reports and/or presentations to assist projects with feature design, prioritization, UX etc.
+
+#### Community Enablement
+
+    * SIGs are open organizations with meetings, meeting agendas and notes, mailing lists, and other communications in the open
+
+    * The mailing list, SIG meeting calendar, and other communication documents of the SIG will be openly published and maintained
+
+#### As Trusted Expert Advisors to the TOC
+
+    * Perform technical due diligence on new and graduating projects, and advise TOC on findings.
+
+    * Be involved with, or periodically check in with projects in their area, and advise TOC on health, status and proposed actions (if any) as necessary or on request.
+
+#### SIG Charter:
+
+    * formally reviewed annually, and approved by the TOC.  The charter must clearly articulate:
+
+        * what is in and out of scope of the SIG
+
+        * whether and how it overlaps and interfaces with other CNCF SIG’s or other relevant groups.
+
+        * how it operates and is governed, and specifically whether and how it deviates from standard SIG operating guidelines provided by the TOC.  Deviation from these guidelines is discouraged, unless there are good and well-documented reasons for such divergence, approved by the TOC. 
+
+See [Example Responsibilities of a CNCF SIG](https://docs.google.com/document/d/1L9dJl5aBFnN5KEf82J689FY0UtnUawnt9ooCq8SkO_w/edit?usp=sharing).
+
+## Operating Model
+
+*Important*
+
+*Each SIG is supported by a named member of the CNCF executive staff who is accountable for liaison with the CNCF Exec, plus communication and performance of the SIG, with quarterly and annual reporting to GB & TOC. *** **
```

Comment:
Check the formatting of this line - the stars might not be doing you expected.

Reply:
"CNCF Exec" ? Do you mean "CNCF Executive Staff" ? Or do you mean "Executive Director" ?  Let's spell out GB since this is the first use of the acronym.

Reply:
Yes, done.